docs/user-guide/cli-using-global-profile-configuration.md

```diff
@@ -79,7 +79,17 @@ You have two options to initialize the global configuration:
 
 You can generate a *user-specific* configuration file that overrides the values defined in the global `zowe.config.json` file.
 
-To generate both global (`zowe.config.json`) and user (`zowe.config.user.json`) config files, issue the command `zowe config init --global-config --user-config`.
+To generate the global profile configuration file (`zowe.config.json`), issue the following command:
+
+```
+zowe config init --global-config
+```
+
+To generate the user profile configuration file (`zowe.config.user.json`), issue the following command:
```

Comment:
Can we call it the "global user profile configuration file", to distinguish it from the project user (non-global) one?